### ISSUE TITLE
Escape special characters in TSV

### ISSUE THREAD

**Issue by jamesaoverton** [knotation/knotation-cljc#16]
In TSV files, literals should escape newlines, tabs, and pipes with a slash.

**Comment by rctauber** [comment]
Are we assuming these literals are in quotes?

**Comment by jamesaoverton** [comment]
No. In a proper TSV file, cell values are not quoted, and quotation marks are not special characters.
